src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./images/Avenir-Black.ttf") format("ttf");
+  font-weight: 300;
+}
+
+.label-search {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 0 10px #f8f8f9;
+  display: block;
+}
+
+.input-area {
+  border: none;
+  outline: none;
+  font-family: Avenir, sans-serif;
+  background-image: url("images/Search.svg");
+  background-repeat: no-repeat;
+  background-color: transparent;
+}
+
+.input-big {
+  padding-left: 62px;
+  padding-bottom: 23px;
+  padding-top: 25px;
+  background-size: 19px;
+  background-position: 26px;
+}
+
+.input-small {
+  padding-left: 33px;
+  padding-bottom: 10px;
+  padding-top: 12px;
+  background-size: 11px;
+  background-position: 13px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.
**You should have input and input--small or input--big, but not input--small and input--big. Some of those inputs is default**